dts/bindings/sensor/maxim,max31865.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022, HAW Hamburg FTZ-DIWIP
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: |
+  Maxim MAX31865 SPI RTD-to-Digital Converter Temperature Sensor.
+  Find the datasheet here:
+  https://datasheets.maximintegrated.com/en/ds/MAX31865.pdf
+
+compatible: "maxim,max31865"
+
+include: spi-device.yaml
+
+properties:
+
+  resistance-at-zero:
+        type: int
+        default: 100
+        required: false
+        description: Sensor resistance at 0 Celsius (100 for PT100, 1000 for PT1000)
+
+  resistance-reference:
+        type: int
+        default: 430
+        required: false
+        description: Circuit reference resistance (recommended on MAX31865 datasheet 400 ohms for PT100, 4000 for PT1000)
+
+  low-threshold:
+        type: int
+        default: 0
+        required: false
+        description: Low fault threshold (ADC CODE)
+
+  high-threshold:
+        type: int
+        default: 32767
+        required: false
+        description: High fault threshold (ADC CODE)
+
+  three-wire:
```

Comment:
use `maxim,3-wire` as property name to match "upstream" linux binding:

https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/iio/temperature/maxim,max31865.yaml